### DISCUSSION TITLE
buddhism/2024/heart_sutra/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#70] [General]
# buddhism/2024/heart_sutra/

https://shanenull.com/buddhism/2024/heart_sutra/

**Comment by shane0** [comment]
# study

- from the large prajnaparamita

> Śāradvatīputra, emptiness [F.50.b] neither arises nor ceases. It is neither afflicted nor purified. It neither decreases nor increases. It is neither past, nor future, nor present. In that which neither arises nor ceases, is neither afflicted nor purified, neither decreases nor increases, and is neither past, nor future, nor present, there are no physical forms, no feelings, no perceptions, no formative predispositions, and no consciousness. In that, there are no eyes, no ears, no nose, no tongue, no body, no mental faculty, no sights, no sounds, no odors, no tastes, no tangibles, and no mental phenomena.
“In that, there is no earth element, no water element, no fire element, no wind element, no consciousness element, and no space element. In that, there is no sensory element of the eyes, no sensory element of sights, no sensory element of visual consciousness, no sensory element of the ears, no sensory element of sounds, no sensory element of auditory consciousness, no sensory element of the nose, no sensory element of odors, no sensory element of olfactory consciousness, no sensory element of the tongue, no sensory element of tastes, no sensory element of gustatory consciousness, no sensory element of the body, no sensory element of touch, no sensory element of tactile consciousness, no sensory element of the mental faculty, no sensory element of mental phenomena, and no sensory element of mental consciousness.
“In that, there is no ignorance, no cessation of ignorance, no formative predispositions, no cessation of formative predispositions, no consciousness, no cessation of consciousness, no name and form, no cessation of name and form, no six sense fields, no cessation of the six sense fields, no sensory contact, no cessation of sensory contact, no sensation, [F.51.a] no cessation of sensation, no craving, no cessation of craving, no grasping, no cessation of grasping, no rebirth process, no cessation of the rebirth process, no actual birth, no cessation of actual birth, {Dt.47} no aging and death, and no cessation of aging and death.
“In that, there is no understanding of suffering, its origins, its cessation, and the path.
“In that, there is nothing to be attained. There is nothing to be clearly realized. In that, there is no one entering the stream to nirvāṇa, no fruit of entering the stream to nirvāṇa, no one destined for only one more rebirth, no fruit of being destined for only one more rebirth, no one who is no longer subject to rebirth, no fruit of not being any longer subject to rebirth, no arhat, no fruit of arhatship, no individual enlightenment, and no pratyekabuddha. In that, there is no bodhisattva and no knowledge of the path. In that, there is no buddha and no enlightenment.

## high or low level high or low detail

- emptiness with no relationships no reference in space or time in out in between small or large
- https://shanenull.com/buddhism/2023/heart_sutra_details/

**Reply by shane0** [reply]
https://en.wikipedia.org/wiki/Heart_Sutra

**Reply by shane0** [reply]
https://plumvillage.org/about/thich-nhat-hanh/letters/thich-nhat-hanh-new-heart-sutra-translation

**Reply by shane0** [reply]
Red pine comments
https://northwestdharma.org/red-pine-on-heart-sutra-for-seattle-ecosangha/

**Reply by shane0** [reply]
# bdk commentary

#113

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/shunryu/#emptiness-is-form

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/dukkha/

**Reply by shane0** [reply]
ct translation read

https://youtu.be/Gke7Igwv1Xg

**Reply by shane0** [reply]
Sheng
https://youtu.be/NA_4S8uDYQ4?si=tDtmLsYXhRw36VhV

**Reply by shane0** [reply]
## A

https://www.samyetranslations.org/wp-content/uploads/2024/03/Perfection-of-Wisdom-in-One-Syllable-English.pdf

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/heart_sutra/

**Comment by shane0** [comment]
# Namo 

is a Pali word that means "homage," "reverence," or "salutation." It is a common Buddhist term used to express respect and devotion to the Buddha, the Dharma (teachings of the Buddha), and the Sangha (community of Buddhist monks and nuns).

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/skhandas/

**Comment by shane0** [comment]
https://github.com/shane0/buddhism/issues/32

**Comment by shane0** [comment]
https://youtu.be/LUjqAajuvZk?si=yOCpwPrg8A8JdNWP
Summary of Diamond summary of maha prajna paramita

**Comment by shane0** [comment]
Playlist
https://youtube.com/playlist?list=PLqhBSEbitutu1nltHWUPyLzgRJx0VLRbL&si=fyEIIndnom1PZ3Yy

**Comment by shane0** [comment]
https://youtu.be/ej0_39J4yts?si=x68KkYlOZ_esll66
Llama

**Comment by shane0** [comment]
breaks down 5 step level

- gate go path of accumulation 
- gate go beyond 
- paragate go utterly beyond - see
- paragate path of meditation
- bodhi svaha - fully awaken no more learning

bodhisatva unconditional unconditioned compassion drives shunyata

https://youtu.be/gxQL4_DBF1w?si=Y9gPvbN8W6B9uK54

**Comment by shane0** [comment]
Free epub
https://www.buddhisttexts.org/products/the-heart-of-prajna-paramita-sutra

**Comment by shane0** [comment]
bdk
https://www.bdkamerica.org/product/a-comprehensive-commentary-on-the-heart-sutra/
https://www.bdkamerica.org/product/the-diamond-needle-treatise-on-the-doctrinal-distinctions-of-the-huayan-one-vehicle-the-key-to-the-secret-of-the-heart-sutra/

**Reply by shane0** [reply]
https://www.bdkamerica.org/wp-content/uploads/2023/09/dBET_DiamondNeedle_HuayanOneVehicle_SecretOfTheHeartSutra.pdf

**Comment by shane0** [comment]
The Sanskrit name for the Heart Sutra is **Prajñāpāramitāhṛdaya**.

* **Prajñāpāramitā** translates to "Perfection of Wisdom"
* **Hṛdaya** translates to "Heart"

So, the whole title translates to "The Heart of the Perfection of Wisdom".

**Comment by shane0** [comment]
THE HEART OF PRAJNA PARAMITA SUTRA
Translated by the Chung Tai Translation Committee
from the Chinese translation by Tripitaka Master Kuanzang of the 7th
century
Bodhisattva Avalokiteshvara,
while deeply immersed in prajna paramita,
clearly perceived the empty nature of the five skandhas, and transcended all suffering.
Shariputra! Form is not different from emptiness, emptiness is not different from form.
Form is emptiness, emptiness is form.
So it is with feeling, conception, volition, and consciousness.
Shariputra! All dharmas are empty in character; neither arising nor ceasing, neither impure nor pure, neither increasing nor decreasing.
Therefore, in emptiness, there is no form;
there is no feeling, conception, volition, or consciousness;
no eye, ear, nose, tongue, body, or mind; no form, sound, smell, taste, touch, or dharmas;
no realm of vision, and so forth, up to no realm of mind-consciousness;
no ignorance or ending of ignorance, and so forth, up to no aging and death or ending of aging and death.
There is no suffering, no cause, no extinction, no path.
There is no wisdom and no attainment.
There is nothing to be attained By way of prajna paramita,
the bodhisattva's mind is free from hindrances.
With no hindrances, there is no fear; freed from all distortion and delusion, ultimate nirvana is reached.
By way of prajna paramita,
buddhas of the past, present, and future attain anuttara-samyak-sambodhi.
Therefore, prajna paramita is the great powerful mantra, the great enlightening mantra, the supreme and peerless mantra.
It can remove all suffering.
This is the truth beyond all doubt.
And the prajna paramita mantra is spoken thus: gate gate paragate parasamgate bodhi svaha.

**Comment by shane0** [comment]
my tts https://youtu.be/cRicqtjDylw
similar version
https://youtu.be/eSmHTKn21vI

**Comment by shane0** [comment]
variation in translation
https://youtu.be/9oflhxoI3ew?si=0i7HwQFMBMbiME74

**Comment by shane0** [comment]
# fearless 

Kua refers to the obstruction of the afflictions (klesavarana), which prevents one from reaching nirvana; ai refers to the obstruction of the known (jñeyavarana), which prevents one from attaining enlightenment. Or, it may be said that kua is ai, for both denote the two obstructions [of affliction and of the known].
Fear is of five kinds: (1) fear of mortality, arising from the craving for one's existence; (2) fear of infamy, arising from not benefiting others and from having desires; (3) fear of death, arising from having the thought of the destruction of one's self; (4) fear of evil transmigrations, arising from not meeting with the Buddhas and from evil karma; and (5) fear of others, arising from seeing oneself as inferior and others as superior.

**Reply by shane0** [reply]
Commentary book page 116

**Reply by shane0** [reply]
Page 107 links

https://github.com/shane0/buddhism/issues/104

**Comment by shane0** [comment]
# abbreviation

First letter of each word


wdiipp ptenotfs atas sfindfe eindff fie eif
siifevac sadaeic nanc ninp nind tie tinf tinfcvoc
neentbom nssttod nrovasf utnrom nioeoiasf
utnaadoeoaad tinsncnenp tinwana tintbabwopp
tbmiffh wnhtinf ffadadunir bwoppbotppaf aa tppitgpm
tgem tsapm icras tittbad atppmist ggppbs

**Comment by shane0** [comment]
https://www.instagram.com/reel/C1ZfiqTRkK8/?igsh=MWpneGhyZjVkNm12dg==

**Comment by shane0** [comment]
# first letter of each sentence

bwcasefessnnntttnnnunuttttwfbatttitag

**Comment by shane0** [comment]
| Sanskrit Term | English Translation | Description |
|---|---|---|
| Rupa | Form | The physical form of an object or experience. |
| Vedana | Feeling | The pleasant, unpleasant, or neutral feeling tone of an experience. |
| Sañjña | Perception* | The recognition of an object or experience. |
| Samskara | Conception/Volition | Mental formations or dispositions that arise from past experiences. |
| Vijnana | Consciousness | Awareness of the present moment. |

*Note: "Sañjña" is the standard Sanskrit term for the third skandha, often translated as "perception."

**Comment by shane0** [comment]
# The "Heart Sutra of the Perfection of Wisdom"

(Panruo Boluomiduo Xin Jing 般若波羅蜜多心經; Prajñāparāmitā-Hridaya-Sūtra),
translated into Chinese by Xuanzang 玄奘

(Taishō Ｔ０２５１)

觀自在菩薩。行深般若波羅蜜多時。照見五蘊皆空。度一切苦厄。舍利子，色不異空。空不異# 色。色即是空。空即是色。受、想、行、識，亦復如是。舍利子，是諸 法空相。不生，不滅。不垢，不淨，不增，不減。是故空中。無色。無受、想、行、識。無眼、耳、鼻、舌、身、意。無色、聲、香、味、觸、法。無眼界。乃至無 意識界。無無明。亦無無明盡。乃至無老死。亦無老死盡。無苦、集、滅、道。無智，亦無得。以無所得故。菩提薩埵。依般若波羅蜜多故。心無罣礙。無罣礙故。 無有恐怖。遠離顛倒夢想。究竟涅槃。三世諸佛。依般若波羅蜜多故。得阿耨多羅三藐三菩提。故知般若波羅蜜多。是大神咒。是大明咒是無上咒。是無等等咒。能 除一切苦。真實不虛故。說般若波羅蜜多咒即說咒曰：揭帝，揭帝，般羅揭帝，般羅僧揭帝，菩提僧莎訶。

When Bodhisattva Avalokiteshvara was practicing the profound Perfection of the Wisdom (prajñā parāmitā), he illuminated the Five Appearances (skandha) and saw that they are all empty (shunya), and he crossed beyond all suffering and difficulty.

Shāriputra, form does not differ from emptiness; emptiness does not differ from form. Form itself is emptiness; emptiness itself is form. So too are feeling, cognition, formation, and consciousness.

Shāriputra, all teachings (dharma) are empty of characteristics. They are not produced, not destroyed, not defiled, not pure; and they neither increase nor diminish. 

Therefore, in emptiness there is no form, feeling, cognition, formation, or consciousness; no eyes, ears, nose, tongue, body, or mind; no sights, sounds, smells, tastes, objects of touch, or dharmas; no field of the eyes up to and including no field of mind consciousness; and no ignorance or ending of ignorance, up to and including no old age and death or ending of old age and death. 

There is no suffering, no accumulating, no extinction, and no Way, and no understanding and no attaining.

Because nothing is attained, the Bodhisattva through reliance on prajñā parāmitā is unimpeded in his mind. 

Because there is no impediment, he is not afraid, and he leaves distorted dream-thinking far behind. 

Ultimately Nirvana! 

All Buddhas of the three periods of time attain anuttarā-samyak-sambodhi through reliance on prajñā parāmitā. 

Therefore know that prajñā parāmitā is a Great Spiritual Mantra, a Great Bright Mantra, a Supreme Mantra, an Unequalled Mantra. 

It can remove all suffering; it is genuine and not false. 

That is why the Mantra of Prajna Paramita was spoken. 

Recite it like this: Gaté Gaté Paragaté Parasamgaté. Bodhi Svaha!
